EKF/vel_pos_fusion.cpp

```diff
@@ -172,6 +172,7 @@ void Ekf::fuseVelPosHeight(const float innov, const float innov_var, const int o
 	// if the covariance correction will result in a negative variance, then
 	// the covariance matrix is unhealthy and must be corrected
 	bool healthy = true;
+	setVelPosFaultStatus(obs_index, false);
```

Comment:
Good point, this could have been causing issues if for some values P < KHP but not the last one...